### PR TITLE
docs: api reference intro

### PR DESCRIPTION
**Pull request by stefanorosanelli**
New API reference intro
Something important missing?
Clear enouhg?
Typos?

### CODE REVIEW

File: source/endpoints/intro.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+Headers
+-------
+
+Since the primary exchange format is JSON-API its `content negotiation rules <http://jsonapi.org/format/#content-negotiation>`_
+are applied with some extensions.
+
+Every request must contain an ``Accept`` header with one of the following values:
+
+    * ``application/vnd.api+json`` JSON-API default content type
+    * ``application/json`` is also fine; considered a synomym for JSON-API, is treated the same way
```
Comment (didoda) [inline source/endpoints/intro.rst]: "synonym"

File: source/endpoints/intro.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+Under special circumstances, and only for GET requests, also ``text/html``, ``application/xhtml+xml``, ``application/xhtml``
+and ``text/xhtml`` may be valid provided that HTML accept is configured (see :ref:`configuration-accept`):
```
Comment (didoda) [inline source/endpoints/intro.rst]: "[...] may be valid, provided that HTML accept is configured" or "[...] may be valid, if HTML accept is configured"

File: source/endpoints/intro.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+response will be an HTML version of the JSON body.
+Main use is enable data navigation using a browser only in test and development enviroments.
+
+Request containing input data like POST, PATCH and in some cases DELETE must also provide a ``Content-Type`` header
+where accepted values are:
+
+    * ``application/vnd.api+json`` JSON-API default content type
+    * ``application/json`` is considered a synomym for JSON-API just like ``Accept``
```
Comment (didoda) [inline source/endpoints/intro.rst]: "synonym"

**Review by didoda — CHANGES_REQUESTED** [review]
minor: typos fixes requested

File: source/endpoints/intro.rst (hunk 1 above)
Comment (stefanorosanelli) [reply source/endpoints/intro.rst]: done

File: source/endpoints/intro.rst (hunk 2 above)
Comment (stefanorosanelli) [reply source/endpoints/intro.rst]: done

File: source/endpoints/intro.rst (hunk 3 above)
Comment (stefanorosanelli) [reply source/endpoints/intro.rst]: done

**Review by didoda — APPROVED** [review]

**Pull request merged**